README.rst

```diff
@@ -19,29 +19,6 @@
 |TatSu|
 =======
 
-.. code-block:: python
```

Comment:
I liked this banner :-\

Reply:
I can bring it back, if you want. Because there will be releases covering most of the active Python versions, I thought the Python version compatibility notice does not require such prominent visibility.